transformer/transformers/ampruntimejs.go

```diff
@@ -0,0 +1,35 @@
+package transformers
+
+import (
+	"net/url"
+	"strings"
+
+	"github.com/ampproject/amppackager/transformer/internal/amphtml"
+	"github.com/ampproject/amppackager/transformer/internal/htmlnode"
+	"golang.org/x/net/html"
+	"golang.org/x/net/html/atom"
+)
+
+// AMPRuntimeJS rewrites the value of src in script nodes, where applicable.
+// If the value is of the form "*.js", replace it with "*.js?f=sxg".
+func AMPRuntimeJS(e *Context) error {
+	for n := e.DOM.HeadNode; n != nil; n = n.NextSibling {
```

Comment:
Ahh, I see what's happened now. We forgot to start at the `<head>`'s child:

```suggestion
	for n := e.DOM.HeadNode.FirstChild; n != nil; n = n.NextSibling {
```

Reply:
Done.